pkg/apis/networking/v1alpha1/service_types.go

```diff
@@ -72,6 +72,7 @@ type ExposurePort struct {
 	Name string `json:"name,omitempty"`
 
 	// Port specifies the exposed service port.
+	// If not set, all ports of the referencing service will be selected.
```

Comment:
We have already explained here that all ports are exposed by default. Do we need to reiterate this here?

https://github.com/karmada-io/karmada/blob/ccd8c3f863401ecf00bc8589e476bd5169d43d5f/pkg/apis/networking/v1alpha1/service_types.go#L41

I understand your concerns. You may be worried that users haven't set both the name and port, but is it allowed to configure YAML like this?

Reply:
> I understand your concerns. You may be worried that users haven't set both the name and port, but is it allowed to configure YAML like this?

Yes.

> We have already explained here that all ports are exposed by default. Do we need to reiterate this here?


ooooh, Thank you for the reminder, I didn't notice it here.

Reply:
I believe that describing this field separately can help users understand it more quickly.